newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGameLink.js

```diff
@@ -350,6 +350,11 @@ const OnlineGameLink = ({
               actions={dialogActions}
               open
               onRequestClose={() => setIsShareDialogOpen(false)}
+              onApply={() => {
+                if (game && buildUrl && !isBuildPublished) {
+                  setIsOnlineGamePropertiesDialogOpen(true);
```

Comment:
Dismiss the dialog will open a new one, only if `game && buildUrl && !isBuildPublished` is true?
This behaviour may feel a bit weird

Reply:
In this case, the button to publish on Liluo.io is only visible if this is a game you own + this is a new build (not published yet)
This button opens a new modal to publish this build (with game details summary)
The button is a primary button, so I think it makes sense to be able to click on it with a shortcut?